src/about.py

```diff
@@ -61,7 +68,15 @@ def __init__(self, window):
         '\n\n' +
         '<small>Copyright © 2005-2009 Pontus Ekberg\n\n' +
         'herrekberg@users.sourceforge.net\n' +
-        'http://comix.sourceforge.net</small>\n')
+        'http://comix.sourceforge.net</small>\n'
+        '<small>Copyright © 2010-2017 David Pineau\n\n' +
+        'dav.pineau@gmail.com\n' +
+        'https://github.com/Joacchim/Comix</small>\n'
+        '<small>Copyright © 2014-2017 Sergey Dryabzhinsky\n\n' +
+        'sergey.dryabzhinksy@gmail.com\n' +
+        'https://github.com/sergey-dryabzhinksy/Comix</small>\n'
```

Comment:
Well, I gave the link to each fork, but I can only leave mine if you think this is better, I'm unsure what would be better here (since I originally left the historical link for comix repository on sourceforge)